### DISCUSSION TITLE
Noice log in seperate buffer with live reload

### DISCUSSION THREAD

**Discussion by azhahes** [LazyVim/LazyVim#1403] [Q&A] [answered]
I was writing some lua script and when i try to run I want to see some log. But since in lazyvim distro, the messages are shown for short amount of time. I want to see the messages in split buffer without doing `:Noice` each time. Is there anyway to do that? can someone help me here?

**Comment by dpetka2001** [comment]
There is already an open issue at `noice` repo https://github.com/folke/noice.nvim/issues/493. You might want to follow that to see if there will be any progress regarding this.

**Reply by jyuan0** [reply, marked as answer]
To add to this, and although this doesn't have live reload, you can configure Noice to route certain messages to a split:
```lua
-- always route any messages with more than 20 lines to the split view
require("noice").setup({
  routes = {
    {
      view = "split",
      filter = { event = "msg_show", min_height = 20 },
    },
  },
})
```

https://github.com/folke/noice.nvim#-routes

Hope this helps.

**Reply by azhahes** [reply]
Thanks @jyuan0 .. That helps!!
